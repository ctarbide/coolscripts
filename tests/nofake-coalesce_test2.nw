
case 2

<<chunk a>>=
1
@
<<chunk a>>=
2
@
