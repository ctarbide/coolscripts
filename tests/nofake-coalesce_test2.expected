
case 2

<<chunk a>>=
1
2
@
